bin/build-tutorials

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+BUILD_TOOL='./tools/codelabs'
+GDOC_IDS_FILE='config/google-documents'
+
+while read ID; do
+  ${BUILD_TOOL} add "${ID}"
+done <"${GDOC_IDS_FILE}"
```

Comment:
Could we do: 

`done < "${GDOC_IDS_FILE}" | sed -e '/^\s*$/ d' -e '/^#/ d'`

So that we can add blank lines and comments (eg. title, date, ...) to the `google-documents` file to ease maintenance?